src/modules/auth/auth.service.ts

```diff
@@ -117,4 +117,34 @@ export class AuthService {
     });
     return authToken;
   }
+
+  async googleLogin(req: any) {
+    if (!req.user) {
+      return { message: 'No user from Google' };
```

Comment:
It would be better to check `req.user` in the controller before calling the service (throwing an exception if fails) and pass the user object itself to the service instead of the entire request object. 
`async googleLogin(requestUser: IRequestUser) {...`